CHANGELOG.md

```diff
@@ -4,6 +4,54 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
+
+## Unreleased - as of Sprint 68 ending 2017-09-04
+
+### Added
+- Internationalization
+  - i8N: Add Data Types to dictionary [(#15922)](https://github.com/ManageIQ/manageiq/pull/15922)
```

Comment:
Sorry, I just saw this.  Is `i18N` redundant since the section is already Internationalization?

I don't recall ever seeing i18N or I18n, only i18n or I18N.

At least we're consistently inconsistent:

```
03:56:40 ~/Code/manageiq (master) (2.4.2) + git grep -i i18n CHANGELOG.md
CHANGELOG.md:  - i18n support in pdf reports
CHANGELOG.md:  - i18n support for UI plugins
CHANGELOG.md:  - I18n support added to the Self Service UI
CHANGELOG.md:- I18n
CHANGELOG.md:  - I18n for toolbars
CHANGELOG.md:  - Fixes for Japanese I18n support
CHANGELOG.md:- I18N
CHANGELOG.md:  - HAML and I18n strings 100% completed in views
CHANGELOG.md:### I18n
CHANGELOG.md:  - All strings in the views have been converted to use gettext (I18n) calls
CHANGELOG.md:  - Can add/update I18n files with translations
```

I can fix in a followup if there's consensus.   It seems to be consistently either both lowercase or both uppercase [here](https://www.w3.org/International/questions/qa-i18n).

Reply:
Good catch,..let me clean it up.